airbyte-integrations/connectors/source-goldcast/source_goldcast/source.py

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from airbyte_cdk.sources.declarative.yaml_declarative_source import YamlDeclarativeSource
+
+"""
+This file provides the necessary constructs to interpret a provided declarative YAML configuration file into
+source connector.
+
+WARNING: Do not modify this file.
+"""
+
+
+# Declarative Source
+class SourceGoldcast(YamlDeclarativeSource):
```

Comment:
In theory you can drop this file and just inherit from YamlDeclarativeSource in `main.py`, but it's okay either way